UnitySDK/Assets/ML-Agents/Scripts/Agent.cs

```diff
@@ -246,6 +236,20 @@ public AgentInfo Info
 
         WriteAdapter m_WriteAdapter = new WriteAdapter();
 
+
+        /// Represents the reward the agent accumulated during the current step.
+        /// It is reset to 0 at the beginning of every step.
```

Comment:
```suggestion
        /// It is reset at the beginning of every step.
```

Reply:
I think this comment block should be rewritten, and probably belongs on the `rewardProvider` property instead.